client/src/client-only-routes/show-user.tsx

```diff
@@ -85,11 +85,9 @@ function ShowUser({
       <main>
         <FullWidthRow>
           <Spacer size='large' />
-          <Panel bsStyle='info' className='text-center'>
+          <Panel varient='primary' className='text-center'>
```

Comment:
needed to change it, because the styles in the info here is reverted than how we decided the info to look, which made it hard to read.